docs/src/components/CardLink.tsx

```diff
@@ -1,20 +1,80 @@
-import { Link } from '@aws-amplify/ui-react';
+import classNames from 'classnames';
+import { Grid, Link, Heading } from '@aws-amplify/ui-react';
 
 interface CardLinkProps {
   href: string;
   title: string;
   desc: string;
-  img: React.ReactNode;
+  icon?: React.ReactNode;
+  className?: string;
+  variation?: 'plain' | 'branded' | 'home';
 }
 
-export function CardLink({ href, title, desc, img }: CardLinkProps) {
+export function CardLink({
+  href,
+  title,
+  desc,
+  icon,
+  variation = 'plain',
+  className,
+}: CardLinkProps) {
+  const classes = classNames(
+    `docs-cardLink`,
+    `docs-cardLink--${variation}`,
+    className
+  );
+
   return (
-    <Link href={href} className="docs-cardLink">
-      {img && <div className="docs-cardLink-img">{img}</div>}
+    <Link href={href} className={classes}>
+      {icon && <div className="docs-cardLink-img">{icon}</div>}
```

Comment:
Do we want to rename the class name to something like `docs-cardLink-icon`?

Reply:
Yes, updated in latest!